app/src/main/java/org/eyeseetea/malariacare/views/question/IValidationQuestion.java

```diff
@@ -0,0 +1,7 @@
+package org.eyeseetea.malariacare.views.question;
+
+public interface IValidationQuestion {
```

Comment:
this interface only is used in a base class CommonQuestionView, then I think is not necessary